evident/tests/support_files/simulated_data_setting.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## This notebook is to simulate the data (mapping files and alpha diversity) for testing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import click \n",
+    "import joblib \n",
+    "import pickle \n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "from os import mkdir\n",
+    "from os.path import join, isdir, basename, exists\n",
+    "from itertools import combinations\n",
+    "from operator import itemgetter\n",
+    "from functools import partial\n",
+    "from skbio import DistanceMatrix\n",
+    "from skbio.stats.distance import permanova\n",
+    "from scipy.stats import mannwhitneyu, spearmanr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## simulated mapping files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# general mapping file: 120 subjects with 20 missing values \n",
+    "# age -- continous variable; \n",
+    "# gender -- categorical variable with 2 levels; \n",
+    "# country -- categorical variable with 4 levels\n",
+    "np.random.seed(31)\n",
+    "mapping = pd.DataFrame({'#SampleID': np.arange(start=1, stop=121, step=1),\n",
```

Comment:
For all the cells here, could you make sure you follow the pep8 80 char rule so it's easier to read?

Reply:
done